#place positions of the carts

cart0_x: 0
cart0_y: 6
cart0_theta: 0
  
cart1_x: 0
cart1_y: 4.5
cart1_theta: 0

cart2_x: 0
cart2_y: 3
cart2_theta: 0

cart3_x: 0
cart3_y: 1.5
cart3_theta: 0

cart4_x: 0
cart4_y: 0
cart4_theta: 0

cart5_x: 0
cart5_y: -1.5
cart5_theta: 0

cart6_x: 0
cart6_y: -3
cart6_theta: 0

cart7_x: 0
cart7_y: -4.5
cart7_theta: 0

cart8_x: 0
cart8_y: -6
cart8_theta: 0

cart9_x: 0
cart9_y: -7.5
cart9_theta: 0